components/Cards.js

```diff
@@ -20,3 +20,92 @@
 // Add a listener for click events so that when a user clicks on a card, the headline of the article is logged to the console.
 //
 // Use your function to create a card for each of the articles, and append each card to the DOM.
+
+import axios from "axios";
+
+const articleCard = (headline, authorPhoto, authorName) => {
+  const card = document.createElement("div");
+  const head = document.createElement("div");
+  const author = document.createElement("div");
+  const imgCont = document.createElement("div");
+  const authPhoto = document.createElement("img");
+  const authName = document.createElement("span");
+
+  card.append(head, author);
+  author.append(imgCont, authName);
+  imgCont.appendChild(authPhoto);
+
+  card.classList.add("card");
+  head.classList.add("headline");
+  author.classList.add("author");
+  imgCont.classList.add("img-container");
+
+  head.textContent = headline;
+  authPhoto.src = authorPhoto;
+  authName.textContent = authorName;
+
+  card.addEventListener("click", () => {
+    card.classList.toggle("card");
+    console.log("card", card);
+  });
+
+  return card;
+};
+
+// articleMaker.appendChild(articleCard);
+
+// articleMaker.appendChild(articleArr);
+
+// console.log(articleMaker);
+
+// axios
+//   .get(`https://lambda-times-api.herokuapp.com/articles`)
+//   .then((success) => {
+//     const nodeList = document.querySelectorAll(".articles");
+//     const articleArr = Array.from(nodeList);
+
+//     success.articles.forEach((articles) => {
+//       const newArticle = articleCard(
+//         articleArr(articles.headline, articles.authorPhoto, articles.authorName)
+//       );
+//       articleMaker.appendChild(newArticle);
+//     });
+//   })
+//   .catch((err) => {
+//     console.log("err", err);
+//   });
+
+// axios
+//   .get(`https://lambda-times-api.herokuapp.com/articles/headline`)
+//   .then((success) => {
+//     success.data.articles.headline.forEach((headline) => {
+//       const articleHead = articleCard(headline);
+//       articleMaker.appendChild(articleHead);
+//     });
+//   });
+
+// axios
+//   .get(`https://lambda-times-api.herokuapp.com/articles/${headline}`)
+//   .then((head) => {
+//     let articleHead = document.querySelector(".cards-container");
+//     articleHead.appendChild(articleCard(head));
+//   });
+
+const articleMaker = document.querySelector(".cards-container");
+
+const nodeList = document.querySelectorAll(".articles");
+const articleArr = Array.from(nodeList);
+
+axios.get(`https://lambda-times-api.herokuapp.com/articles`).then((success) => {
+  success.data.articleArr.articles.forEach((article) => {
```

Comment:
This is close but not quite correct. If you `console.log(success.data)` you will see this data
```
{
    "articles": {
        "javascript": [{
            "id": "c87ec60b-03f3-4aa3-b9b7-aafa9894045a",
            "headline": "ES8: The Next Step in the Evolution of Javascript and What it Means For Your Projects",
            "authorPhoto": "https://tk-assets.lambdaschool.com/08d1372e-e393-47f1-ac44-fcb7d0baf0e2_sir.jpg",
            "authorName": "SIR RUFF'N'STUFF"
        }, {
            "id": "91f72685-dacb-4a18-a69c-cf5ff95813b7",
            "headline": "Type Coercion: Why Does NaN !== NaN? and Other Strange Occurrences",
            "authorPhoto": "https://tk-assets.lambdaschool.com/a9471235-ed71-4b11-ae15-5a4fa1151d30_bones.jpg",
            "authorName": "BONES R. LIFE"
        }, {
            "id": "0d0e3441-5304-4298-afa9-0062bc684807",
            "headline": "When to Rest, When to Spread: Why There Are Two Meanings Behind '...'",
            "authorPhoto": "https://tk-assets.lambdaschool.com/44260ce3-c8f0-4db8-bc1d-9877662fdf96_puppers.jpg",
            "authorName": "PUPPER S. DOGGO"
        }, {
            "id": "897a0272-2b88-45c4-8c87-9d1885ec24f3",
            "headline": "Typescript: Ten Things you Should Know Before Building Your Next Angular Application",
            "authorPhoto": "https://tk-assets.lambdaschool.com/08d1372e-e393-47f1-ac44-fcb7d0baf0e2_sir.jpg",
            "authorName": "SIR RUFF'N'STUFF"
        }],
        "bootstrap": [{
            "id": "8e807024-4c69-41e2-aed0-5006f5a0ce97",
            "headline": "Bootstrap 5: Get a Sneak Peak at all the New Features",
            "authorPhoto": "https://tk-assets.lambdaschool.com/1891c758-b3f4-4ec7-9d88-590bf7c7ceb2_fido.jpg",
            "authorName": "FIDO WALKSALOT"
        }, {
            "id": "61852afb-5689-4ac4-8d89-dca076f9ce9c",
            "headline": "UI Frameworks: A Comparison, Which Made Our List? (Hint: Bootstrap is on it)",
            "authorPhoto": "https://tk-assets.lambdaschool.com/247dd20f-7e32-479b-a861-c3bbe85b5f0c_max.jpg",
            "authorName": "MAX GOODBOYE"
        }, {
            "id": "112e81ea-5b7f-4288-a114-48fbcba96929",
            "headline": "The Hottest New Bootstrap 4 Components Every Developer Needs to Know About",
            "authorPhoto": "https://tk-assets.lambdaschool.com/247dd20f-7e32-479b-a861-c3bbe85b5f0c_max.jpg",
            "authorName": "MAX GOODBOYE"
        }],
        "technology": [{
            "id": "41889af2-6639-49d1-bf81-7256675cf6ce",
            "headline": "AI: What Are the Ethical Ramifications of the Future?",
            "authorPhoto": "https://tk-assets.lambdaschool.com/247dd20f-7e32-479b-a861-c3bbe85b5f0c_max.jpg",
            "authorName": "MAX GOODBOYE"
        }, {
            "id": "a36571d4-9d62-4850-9029-09f4d853689d",
            "headline": "AR or VR: How We Will See the World Through a Digital Lense",
            "authorPhoto": "https://tk-assets.lambdaschool.com/247dd20f-7e32-479b-a861-c3bbe85b5f0c_max.jpg",
            "authorName": "MAX GOODBOYE"
        }, {
            "id": "710a3a06-2079-4787-b72e-9c145e4d9d5b",
            "headline": "Quantum Computing: Where we Are Now, and Where We Will Be Very Soon",
            "authorPhoto": "https://tk-assets.lambdaschool.com/247dd20f-7e32-479b-a861-c3bbe85b5f0c_max.jpg",
            "authorName": "MAX GOODBOYE"
        }],
        "jquery": [{
            "id": "ee35a93d-c05b-4adf-9662-f1d769224d67",
            "headline": "History of Javascript Libraries: from jQuery to React",
            "authorPhoto": "https://tk-assets.lambdaschool.com/44260ce3-c8f0-4db8-bc1d-9877662fdf96_puppers.jpg",
            "authorName": "PUPPER S. DOGGO"
        }, {
            "id": "acb53628-cfb2-43d4-bbe4-2a103969d240",
            "headline": "jQuery Animations: When and Where to Use Them, and Not CSS",
            "authorPhoto": "https://tk-assets.lambdaschool.com/1891c758-b3f4-4ec7-9d88-590bf7c7ceb2_fido.jpg",
            "authorName": "FIDO WALKSALOT"
        }, {
            "id": "9c07f6e8-3d40-454f-8ec5-7813005670ab",
            "headline": "Why I Use jQuery in Every Application I Create and Why You Should Too",
            "authorPhoto": "https://tk-assets.lambdaschool.com/1891c758-b3f4-4ec7-9d88-590bf7c7ceb2_fido.jpg",
            "authorName": "FIDO WALKSALOT"
        }],
        "node": [{
            "id": "8782058c-94a2-49fb-a2b6-306c2a2586c8",
            "headline": "Node.js in 2018: Which Framework? Express, Sails, Hapi?",
            "authorPhoto": "https://tk-assets.lambdaschool.com/08d1372e-e393-47f1-ac44-fcb7d0baf0e2_sir.jpg",
            "authorName": "SIR RUFF'N'STUF"
        }, {
            "id": "f7424991-24c5-4291-9fa9-5db80fb32ba0",
            "headline": "MongoDB: NoSQL vs. SQL, the Debate Continues..",
            "authorPhoto": "https://tk-assets.lambdaschool.com/a9471235-ed71-4b11-ae15-5a4fa1151d30_bones.jpg",
            "authorName": "BONES R. LIFE"
        }]
    }
}
```